### DISCUSSION TITLE
CMS: Neuen Dienst einrichten > Abfragen nicht mehr konfigurierbar

### DISCUSSION THREAD

**Discussion by k-placer** [e-netze/webgis-community#121] [General]
1. Ich richte einen neuen ArcGIS Server Dienst ein
2. das Unterverzeichnis Services\ArcGisServer\MapServer\test\Queries wird korrekt erzeugt, 
3. in der der CMS-Weboberfläche sehe ich den Link "Abfragen" nicht (mehr):
<img width="283" alt="2023-11-10 10_12_39-webgis - Cms" src="https://github.com/gis-eni/webgis-forum/assets/97947925/b3a526b8-6ab0-4048-aae2-48fd1f568a34">

Hier ein Screenshot von dem Service Test Edit, der vor längerer Zeit eingerichtet wurde. Hier gibt es den Link zu den "Abfragen":
![2023-11-10 10_12_39-webgis - Cms- Abfragen](https://github.com/gis-eni/webgis-forum/assets/97947925/041a2b60-568b-4533-8b24-146c37c97785)

Der verwendete Build ist Version: 5.23.4501

**Comment by fanie-mak** [comment]
Hallo,

wenn du einen neuen Dienst einbaust, werden in den Einstellungen die Abfragen und Darstellungsvarianten "Automatisch" bereitgestellt. In "Test (Eigenschaften)", ganz oben, kannst du das bearbeiten...

![image](https://github.com/gis-eni/webgis-forum/assets/79254257/6d85c6c5-e86b-41ee-ac27-b2c006a22871)

LG

**Reply by k-placer** [reply]
Danke für den Tipp! So funktionert es :-)
